server/app/services/export/JsonExporter.java

```diff
@@ -181,6 +182,8 @@ private CfJsonDocumentContext buildSingleApplicationJson(
                 jsonApplication.putString(
                     submitTimePath, dateConverter.renderDateTimeDataOnly(submitTime)),
             () -> jsonApplication.putNull(submitTimePath));
+    jsonApplication.putString(
+        Path.create("submission_status"), applicationJsonExportData.submissionStatus());
```

Comment:
nitpick: I'm open to being convinced I'm wrong, but I don't _love_ `submission_status` for this field name. It sounds to me like it's the application's status in the submission process, and would have values like "in progress", "accepted", "in review" etc, which is not what this is.

Maybe something along the lines of `revision` or `revision_type`? (Although "type" here is kind of odd too... hmm...)

See also the Google AIPs, which use "revision": https://google.aip.dev/162

This also leaves room for a future "revision_set_id` or `application_set_id` field that ties revisions of the same application together

Reply:
`revision_state` could work here actually, instead of `revision_type`

(Inspired by https://google.aip.dev/216)

Reply:
Done